tasks/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -52,10 +51,9 @@ spec:
         - name: shared-dir
           mountPath: /shared
       workingDir: /shared
-      env:
-        - name: TARGET_GH_REPO
-          value: "$(params.targetGHRepo)"
       script: |
+        TARGET_GH_REPO=$(cat "$(params.dataJsonPath)" | jq -r '."targetGHRepo" // "redhat-appstudio/infra-deployments"')
```

Comment:
Nit: `."targetGHRepo"` can be just `.targetGHRepo`. The apostrophes are only needed if there are special characters like `-` in the field name.